docs/notebooks/01_merge.ipynb

```diff
@@ -1,27 +1,505 @@
 {
```

Comment:
<p>I thought it would be nice to have both gencode and ensembl gtf annotation examples.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icbi-lab/atlas_protocol/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>sounds good!</p><p> </p>